### ISSUE TITLE
The scons python path is not properly checked, resulting "WinError 5 occured" when compiling

### ISSUE THREAD

**Issue by JerryVVan** [Nuitka/Nuitka#839]
### Versions
```
Nuitka --version
0.6.8.4
Python: 3.4.0 (v3.4.0:04f714765c13, Mar 16 2014, 19:24:06) [MSC v.1600 32 bit (Intel)]
Executable: C:\Python34\python.exe
OS: Windows
Arch: x86
```
### Problem
I compiled my project with the command below:
`nuitka --mingw64 --windows-disable-console --standalone --show-progress --show-memory --plugin-enable=qt-plugins --plugin-enable=pylint-warnings --recurse-all --output-dir=out --python-for-scons=C:\Users\haowan\AppData\Local\Programs\Python\Python35 programmer.py`

Here's the console output:
```
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\nuitka\__main__.py", line 195, in <module>
    main()
  File "C:\Python34\lib\site-packages\nuitka\__main__.py", line 188, in main
    MainControl.main()
  File "C:\Python34\lib\site-packages\nuitka\MainControl.py", line 749, in main
    result, options = compileTree(main_module=main_module)
  File "C:\Python34\lib\site-packages\nuitka\MainControl.py", line 685, in compileTree
    result, options = runScons(main_module=main_module, quiet=not Options.isShowScons())
  File "C:\Python34\lib\site-packages\nuitka\MainControl.py", line 566, in runScons
    return SconsInterface.runScons(options, quiet), options
  File "C:\Python34\lib\site-packages\nuitka\build\SconsInterface.py", line 299, in runScons
    return subprocess.call(scons_command, shell=False, cwd=source_dir) == 0
  File "C:\Python34\lib\subprocess.py", line 535, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Python34\lib\subprocess.py", line 848, in __init__
    restore_signals, start_new_session)
  File "C:\Python34\lib\subprocess.py", line 1104, in _execute_child
    startupinfo)
PermissionError: [WinError 5] 拒绝访问。
``` 

Someone told me that reboot PC can solve this kind of problem and
 I tried it, but it didn't work.

Could you give me some suggestions for solving this problem, thanks!

**Comment by kayhayen** [comment]
Wait, there are problems that are not resolved by rebooting Windows? ;-)

I think your path for `--python-for-scons` does not exist, does it? I am going to add an error check for it, which should avoid that misleading error. I cannot read the Chinese (?) output, what does it say in English?

Yours,
Kay

**Comment by kayhayen** [comment]
That said, there already is an error check, but it checks for existence. Your path is probably a directory, and not a file. I am going to change the check, you append `\Python.exe` to it and you will probably be good.

**Comment by JerryVVan** [comment]
I modified `--python-for-scons=C:\Users\haowan\AppData\Local\Programs\Python\Python35` to  `--python-for-scons=C:\Users\haowan\AppData\Local\Programs\Python\Python35\python.exe` and it did work.

BTW, that Chinese means Access denied.

Much thanks

**Comment by kayhayen** [comment]
We keep it open until it arrives on stable release.

**Comment by kayhayen** [comment]
Released as part of 0.6.10
